src/main/java/reposense/parser/RepoConfigCsvParser.java

```diff
@@ -51,6 +62,14 @@ protected void processLine(List<RepoConfiguration> results, String[] elements) t
         String ignoreStandaloneConfig = getValueInElement(elements, IGNORE_STANDALONE_CONFIG_POSITION);
         List<CommitHash> ignoreCommitList = CommitHash.convertStringsToCommits(
                 getManyValueInElement(elements, IGNORE_COMMIT_LIST_CONFIG_POSITION));
+        List<Group> groups = Collections.emptyList();
```

Comment:
why is the group field a list? Is it even meant to be a multi-value field?

Reply:
Each repository can have multiple groups, which is extracted from the groups-config.csv file using ```GroupConfigCsvParser``` and stored in this list.